packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelMain.tsx

```diff
@@ -526,10 +620,33 @@ export function TransferPanelMain({
         }
       },
       to: {
-        disabled: true,
-        options: [to],
+        disabled: !toOptions.length,
+        options: toOptions,
+        disabledChainIds: getDisabledOptions(from.chainID),
         value: to,
-        onChange: () => {}
+        otherSelectedNetwork: from,
+        onChange: async network => {
+          // Selecting the same chain or L1 network
+          if (to.chainID === network.chainID || isL1Network(network)) {
```

Comment:
Are we onboard with this UX? The dropdown is potentially allowing the selection of the same `from` L2 network in `to` dropdown as well. (Although it doesn't let it go through, isn't it a little bit weird to even show an option like that?)

![image](https://user-images.githubusercontent.com/7558499/196049924-3870a345-3083-4ba9-96bc-326f1b51fb98.png)

Reply:
This one is for withdrawals, the screenshot is showing the deposit mode. If the same network is selected in `to` in deposit mode it will swap networks. It's implemented [here](https://github.com/OffchainLabs/arb-token-bridge/pull/458/files#diff-5c3d16bc77b54ee0a8c102c0ba868dface6f97de65e6deea8a26751377d66df1R606-R608)